File: app/src/main/java/io/github/golok56/manajemenuang/ui/base/BaseDialog.java
package io.github.golok56.manajemenuang.ui.base;

import android.support.v4.app.DialogFragment;
import android.widget.Toast;

/**
 * @author Satria Adi Putra
 */
public class BaseDialog extends DialogFragment {

    protected void showToast(String msg) {
        Toast.makeText(getActivity(), msg, Toast.LENGTH_SHORT).show();
    }

}

File: app/src/main/java/io/github/golok56/manajemenuang/views/fragment/dialog/BaseDialog.kt
package io.github.golok56.manajemenuang.views.fragment.dialog

import android.support.v4.app.DialogFragment
import android.widget.Toast

/**
 * @author Satria Adi Putra
 */
open class BaseDialog : DialogFragment() {

    fun showToast(msg: String) {
        Toast.makeText(activity, msg, Toast.LENGTH_SHORT).show()
    }

}
